BrainSentimentIndicatorUniverse.cs

```diff
@@ -0,0 +1,179 @@
+        /// </summary>
+        /// <param name="config">Configuration object</param>
+        /// <param name="date">Date of this source file</param>
+        /// <param name="isLiveMode">true if we're in live mode, false for backtesting mode</param>
+        /// <returns>String URL of source file.</returns>
+        public override SubscriptionDataSource GetSource(SubscriptionDataConfig config, DateTime date, bool isLiveMode)
+        {
+            return new SubscriptionDataSource(
+                Path.Combine(
+                    Globals.DataFolder,
+                    "alternative",
+                    "brain",
+                    "sentiment",
+                    "universe",
+                    $"{date:yyyyMMdd}.csv"
+                ),
+                SubscriptionTransportMedium.LocalFile
+            );
+        }
+
+        /// <summary>
+        /// Parses the data from the line provided and loads it into LEAN
+        /// </summary>
+        /// <param name="config">Subscription configuration</param>
+        /// <param name="line">Line of data</param>
+        /// <param name="date">Date</param>
+        /// <param name="isLiveMode">Is live mode</param>
+        /// <returns>New instance</returns>
+        public override BaseData Reader(SubscriptionDataConfig config, string line, DateTime date, bool isLiveMode)
+        {
+            var csv = line.Split(',');
+            var sentiment7Days = decimal.Parse(csv[4], NumberStyles.Any, CultureInfo.InvariantCulture);
+
+            return new BrainSentimentIndicatorUniverse
+            {
+                TotalArticleMentions7Days = int.Parse(csv[2]),
+                SentimentalArticleMentions7Days = decimal.Parse(csv[3], NumberStyles.Any, CultureInfo.InvariantCulture),
+                Sentiment7Days = sentiment7Days,
+                TotalBuzzVolume7Days = !string.IsNullOrWhiteSpace(csv[5])
+                    ? decimal.Parse(csv[5], NumberStyles.Any, CultureInfo.InvariantCulture)
+                    : null,
+                SentimentalBuzzVolume7Days = !string.IsNullOrWhiteSpace(csv[6])
+                    ? decimal.Parse(csv[6], NumberStyles.Any, CultureInfo.InvariantCulture)
+                    : null,
+
+                TotalArticleMentions30Days = int.Parse(csv[7]),
+                SentimentalArticleMentions30Days = decimal.Parse(csv[8], NumberStyles.Any, CultureInfo.InvariantCulture),
+                Sentiment30Days = decimal.Parse(csv[9], NumberStyles.Any, CultureInfo.InvariantCulture),
+                TotalBuzzVolume30Days = !string.IsNullOrWhiteSpace(csv[10])
+                    ? decimal.Parse(csv[10], NumberStyles.Any, CultureInfo.InvariantCulture)
+                    : null,
+                SentimentalBuzzVolume30Days = !string.IsNullOrWhiteSpace(csv[11])
+                    ? decimal.Parse(csv[11], NumberStyles.Any, CultureInfo.InvariantCulture)
+                    : null,
```

Comment:
I would suggest creating a private class to handle this:
```csharp
private decimal? ToDecimalOrNull(string value)
{
    return !string.IsNullOrWhiteSpace(value)
        ? decimal.Parse(value, NumberStyles.Any, CultureInfo.InvariantCulture)
        : null,
}
```
that could be apply to all possible decimal, for those that are never `null`, like `sentiment7Days`, you could use 
```csharp
var sentiment7Days = ToDecimalOrNull(csv[4]).Value,
```